manim/mobject/types/image_mobject.py

```diff
@@ -44,6 +43,9 @@ def reset_points(self):
         )
         self.center()
         h, w = self.get_pixel_array().shape[:2]
+        resolution_of_final_video = 1080
+        self.height = h / resolution_of_final_video * 8.0
```

Comment:
This 8.0 should also be removed, and instead, should come from config["frame_height"]